### ISSUE TITLE
Execute time not showing in Jupyterlab 3.x

### ISSUE THREAD

**Issue by max-mayrhofer** [deshaw/jupyterlab-execute-time#44]
I upgraded from Jupyterlab 2.x to 3.x, and migrated jupyterlab-execute-time as well, but it no longer shows the execute time in the notebooks. Configuration was verified:

![image](https://user-images.githubusercontent.com/12987411/123392014-e182eb00-d59c-11eb-8c3c-854fe5d5d1eb.png)
![image](https://user-images.githubusercontent.com/12987411/123392114-fa8b9c00-d59c-11eb-886a-3474da90230a.png)
![image](https://user-images.githubusercontent.com/12987411/123392148-070ff480-d59d-11eb-9330-a88e31e1fba0.png)
![image](https://user-images.githubusercontent.com/12987411/123392368-43435500-d59d-11eb-861b-534e41f7b323.png)

Any ideas?

**Comment by cherish0** [comment]
same issue

**Comment by mlucool** [comment]
Thanks for reporting. The binder is working and using 3.0.16.

In console, can you confirm if you see a line like this? 
![image](https://user-images.githubusercontent.com/1813603/123643758-b8f42e80-d7f2-11eb-84a5-ff3658c29e3c.png)

**Comment by max-mayrhofer** [comment]
Unfortunately not, no such line :(

**Comment by shivam13juna** [comment]
@mlucool Can you try just changing dependency, that might work, as you say binder is working correctly.

**Comment by mlucool** [comment]
The install in jupyterlab version is the 2.x version, which I agree is confusing. The dependencies are correct in the pip version (which is a prebuilt extension): https://github.com/deshaw/jupyterlab-execute-time/blob/master/package.json#L60

**Comment by max-mayrhofer** [comment]
I saw that, but is the pypi package up to date?

**Comment by YichaoOU** [comment]
having this issue too, is it solved?

**Comment by shivam13juna** [comment]
@YichaoOU no man, waiting on mlucool to update

**Comment by rleyvasal** [comment]
Having the same issue. Using 3.0.14

**Comment by v1lev** [comment]
JupyterLab 3.0.14

I see the line ```jupyterlab-execute-time is activated!``` in console. Extension doesn't work anyway.

**Comment by toddkaufmann** [comment]
I encountered the same  thing when creating a new environment yesterday, so can confirm--
***Doesn't work:***
```
jupyterlab              3.0.16
jupyterlab-execute-time 2.0.4
```
In another environment where I used it recently it was  
**working fine:**
```
jupyterlab              3.0.14
jupyterlab-execute-time 2.0.2
```
So I removed it, installed version 2.0.2, restarted my jupyter server, and now it's working with 3.0.16.  
Other combinations may work as well.

**Comment by mlucool** [comment]
Thanks for letting us know @toddkaufmann. Do you know if either the 2.0.2/3.0.16 or the 2.0.4/3.0.14 combination works?

@goanpeca can you take a look at this? Maybe one of the jupyter_packaging upgrades broke something? We may also want to move binder from a dev install to building + installing from the .whl to be sure it really works.

**Comment by goanpeca** [comment]
Will take a look @mlucool

**Comment by theJukeBoXer** [comment]
I was searching for a solution after upgrading all packages.  Can also confirm that force reinstall of jupyterlab-execute-time 2.0.2 solved the problem for me.
In that in the upgrade process my jupyterlab was inadvertently upgraded to 3.1.0b1.

**Comment by max-mayrhofer** [comment]
Can confirm from my side, have since deployed latest version of jupyterlab, with v2.0.2 of execute time and it works perfectly.

**Comment by mlucool** [comment]
This should really be fixed now in 2.0.5. Please reopen if you run into problems. Sorry for the long delay - turned out to be the publish env and not really the code itself!

**Comment by octavd** [comment]
Hello,

I've tested this plugin - version 2.1.0, 2.0.5 with jupyterlab 3.4.2 and altough it says Jupyterlab extension jupyterlab-execute-time is activated it still didn't work.
However, If you go to settings -> advanced settings editor -> Json settings editor and in the search bar you type recordTiming you will find this has false value

`       // Recording timing
    // Should timing data be recorded in cell metadata
    "recordTiming": false,
`
In my docker image i've put this parameter to true with jq:
`jq '.properties.recordTiming.default = true' /opt/conda/share/jupyter/lab/schemas/@jupyterlab/notebook-extension/tracker.json)" > /opt/conda/share/jupyter/lab/schemas/@jupyterlab/notebook-extension/tracker.json`

After applying the above fix, this extension works.
